container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -27,18 +28,34 @@ const (
 	http_listen {{.Service.HttpServerHost}}
```

Comment:
This string is getting pretty big; maybe worth putting it in its own file and using `go:embed` to get it here for readability?